guides/endpoint.md

```diff
@@ -123,6 +123,26 @@ Path.expand("../../../some/path/to/ssl/key.pem", __DIR__)
 
 If you require further customization to the TLS versions or ciphers used you can include additional `https:` configuration. For example to disable older versions of TLS which are now considered insecure you could add `versions: [:'tlsv1.2']`. More information on the available settings is available in the (Erlang SSL docs)[http://erlang.org/doc/man/ssl.html}] (see "SSL OPTION DESCRIPTIONS - SERVER SIDE").
 
+
+### SSL in Development
+
+If you would like to use HTTPS in development, a self-signed certificate can be generated by running: `mix phx.gen.cert`. This requires Erlang/OTP 20 or later.
+
+With your self-signed certificates, your development configuration in `config/dev.exs` can be updated to run an HTTPS endpoint:
```

Comment:
Should "certificates" be made singular to "certificate"?

Reply:
probably 😛 

fixed